### ISSUE TITLE
Ambiguous occurence of `Type`

### ISSUE THREAD

**Issue by marcosh** [haskell-graphql/graphql-api#150]
Trying to do `stack build graphql-api` in an new project, I get the following error

```
Ambiguous occurrence ‘Type’
        It could refer to either ‘Protolude.Type’,
                                 imported from ‘Protolude’ at src/GraphQL/Internal/Syntax/AST.hs:55:1-16
                                 (and originally defined in ‘GHC.Types’)
                              or ‘GraphQL.Internal.Syntax.AST.Type’,
                                 defined at src/GraphQL/Internal/Syntax/AST.hs:244:1
```

I'm quite a newbie, so I might well be wrong, but it looks like a version problem, maybe somehow related to #121

**Comment by jml** [comment]
Yeah, I had to fix this locally recently. We need to change the `Protolude` import to hide `Type`.

**Comment by jml** [comment]
Fixed by #154
